.github/actions/build-publish-api/action.yml

```diff
@@ -26,8 +26,6 @@ inputs:
 runs:
   using: composite
   steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
```

Comment:
don't need this cause we are already checking out the repo code from the base `deploy-dev.yml` file